wavefront_pyformance/wavefront_reporter.py

```diff
@@ -27,26 +28,41 @@ def __init__(self, source='wavefront-pyformance', registry=None,
         self.prefix = prefix
         self.tags = tags or {}
 
+    @staticmethod
+    def decode_key(key):
+        """Decode encoded key into original key and dict of tags."""
+        if '-tags=' in key:
+            decoded_str = key.split('-tags=')
+            return decoded_str[0], ast.literal_eval(decoded_str[1])
```

Comment:
Any specific reason to use `ast.literal_eval()` instead of `json.loads()` here?

Reply:
Ah, python2 again. Won't `from __future__ import unicode_literals` solve the unicode issues?